### ISSUE TITLE
«bua» vs «bue»

### ISSUE THREAD

**Issue by Ilmen-vodhr** [toaq/dictionary#77]
```
  {
    "toaq": "bua",
    "type": "predicate",
    "english": "▯ inhabits ▯.",
    "gloss": "inhabit",
    "short": "",
    "keywords": [],
    "frame": "c c",
    "distribution": "d d",
    "notes": [],
    "examples": [],
    "fields": []
  },
  {
    "toaq": "bue",
    "type": "predicate",
    "english": "▯ is a home of ▯.",
    "gloss": "home",
    "short": "",
    "keywords": [],
    "frame": "c c",
    "distribution": "d d",
    "notes": [],
    "examples": [],
    "fields": []
  },
```
How do these two differ semantically, irrespectively of the different slot order?
Does ``bủa jí úsōnāgūa`` make sense, or does it mean the whole of the USA is my residence?

**Comment by acotis** [comment]
With **bue** i interpret the home to be a physical structure (like a house or an apartment), and I also interpret it as meaning that the whole physical structure is the home (so eg the whole apartment building is not your bue if you only live in one of the apartments in it). On the other hand, I interpret **bua** as simply saying that you live somewhere within a given area or structure (possibly the whole thing).

**Comment by Ilmen-vodhr** [comment]
Okay. That makes it harder to indicate precisely the extent of the actually inhabited home, though. If you say ``bủe go jí ní jỉo``, it doesn't tell which parts of the building are your actual home; to say it's the whole building, you should then say ``bủe go jí tu pảq nỉ jỉo``, which might not be desirable.

**Comment by acotis** [comment]
No, with my definitions `bủe go jí ní jỉo` does tell you which part of the building is the home: the whole thing. `bủa jí ní jỉo` doesn't tell you that, but I see that as desirable, since it lets you say eg what country you live in without giving your address.

**Comment by Ilmen-vodhr** [comment]
Oh, I've apparently misread part of your previous message, sorry.
With the exhibitor slot rule for serial predicates, one could say ``bủa jí sa pảq nỉ gủe`` to say "I inhabit some part of this country", if it were necessary.
